apps/etterlatte-beregning/src/main/kotlin/beregning/BeregnBarnepensjonService.kt

```diff
@@ -255,19 +255,15 @@ class BeregnBarnepensjonService(
                 )
             }
         },
-        institusjonsopphold = if (featureToggleService.isEnabled(BrukInstitusjonsoppholdIBeregning, false)) {
```

Comment:
Er denne ikke i bruk lenger altså bryteren?

Reply:
Jo, i `beregnBarnepensjon`-metoden for å seie kva regel som brukast.